src/main/resources/config/liquibase/changelog/20210510_transaction_table_update.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+    <changeSet author="LUXIANZE" id="20210510_1">
+        <addNotNullConstraint tableName="transaction" columnName="status" columnDataType="varchar(50)" defaultNullValue="PENDING"/>
+        <addNotNullConstraint tableName="transaction" columnName="amount" columnDataType="decimal(21,2)" defaultNullValue="0.00"/>
+    </changeSet>
+    <changeSet id="20210510_2" author="LUXIANZE">
+        <addColumn tableName="transaction">
+            <column name="title" type="varchar(50)" defaultValue="transaction title">
+                <constraints nullable="false"/>
+            </column>
+            <column name="image_link" type="varchar(255)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="transaction_date" type="datetime" defaultValue="2021-05-10, 00:00:00">
```

Comment:
why default value is given to title and transaction date?
I prefer an exception is thrown instead of we assume the default value

Reply:
Then I need to update the initial fake data?

Reply:
my personal preference will be drop the whole table (means all the table records will be removed as well).
then create a new table, just like what you did, but without defaultValue.

Maybe need to discuss with others, whether defaultValue is acceptable or not.

Reply:
Ok, can discuss tonight

Reply:
Update column to not nullable and provide default value in 1 changeSet, drop default value in another changeSet